src/main/java/TaskManager.java

```diff
@@ -0,0 +1,114 @@
+                + "____________________________________________________________\n";
+        System.out.println(greeting);
+    }
+
+    public void printEndMessage() {
+        String bye = "____________________________________________________________\n"
+                +  " Bye. Hope to see you again soon!\n"
+                + "____________________________________________________________\n";
+        System.out.println(bye);
+    }
+
+    public String[] decodeInput(String input) {
+        return input.split(" ");
+    }
+
+    public void crossOff(String[] inputWords) {
+        if (inputWords.length != 2) {
+            System.out.println(invalidInput);
+            return;
+        }
+        int taskIndex = Integer.parseInt(inputWords[1]) - 1;
+        if (taskIndex >= taskCount) {
+            System.out.println(invalidInput);
+            return;
+        }
+        tasks[taskIndex].setDone();
+        String doneMessage = "____________________________________________________________\n"
+                + " Nice! I've marked this task as done: \n"
+                + " [" + tasks[taskIndex].getStatusIcon() + "] " + tasks[taskIndex].description + "\n"
+                + "____________________________________________________________\n";
+        System.out.println(doneMessage);
+    }
+
+    String description;
+    private void printAdditionMessage() {
+        String addition = "____________________________________________________________\n"
+                + " Gotcha! I've added this task: \n"
+                + "   " + tasks[taskCount - 1].listTask() + "\n"
+                + " Now you have " + (taskCount) + " tasks in the list.\n"
+                + "____________________________________________________________\n";
+        System.out.println(addition);
+    }
+
+    public void addToDo(String input, String[] inputWords) {
+        if (inputWords.length <= 1) {
+            System.out.println(invalidInput);
+            return;
+        }
+        description = input.substring(5);
+        tasks[taskCount] = new ToDo(description);
+        taskCount = taskCount + 1;
+        printAdditionMessage();
+    }
+
+    public void addDeadline(String input, String[] inputWords) {
+        if (!input.contains("/by") || input.endsWith("/by")) {
+            System.out.println(invalidInput);
+            return;
+        }
+        String endDate;
+        description = input.substring(9, input.indexOf("/by") - 1);
+        endDate = input.substring(input.indexOf("/by") + 4);
+        tasks[taskCount] = new Deadline(description, endDate);
+        taskCount = taskCount + 1;
+        printAdditionMessage();
+    }
+
+    public void addEvent(String input, String[] inputWords) {
+        if (!input.contains("/at") || input.endsWith("/at")) {
+            System.out.println(invalidInput);
+            return;
+        }
+        String startEndTime;
+        description = input.substring(6, input.indexOf("/at") - 1);
```

Comment:
6 here seems not meaningful. Try replace it with a constant so that it is easy to read?